### DISCUSSION TITLE
What's the difference between this plugin and `vim.treesitter`?

### DISCUSSION THREAD

**Discussion by benjamin051000** [nvim-treesitter/nvim-treesitter#5248] [Q&A]
I found [this](https://neovim.io/doc/user/treesitter.html), which says:
> By default, Nvim bundles parsers for C, Lua, Vimscript, Vimdoc and Treesitter query files, but parsers can be installed manually or via a plugin like https://github.com/nvim-treesitter/nvim-treesitter.

So, does this plugin simply automate installation of parsers for other langs, but lets `vim.treesitter` do all the actual syntax highlighting work? I was under the impression that this plugin did everything until I noticed that "treesitter" came installed by default. Strange... Anyone have insight into how this works? Thanks!

**Comment by clason** [comment]
> So, does this plugin simply automate installation of parsers for other langs, but lets vim.treesitter do all the actual syntax highlighting work?

That is correct.
